src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/ControllerArgumentValueResolverPassTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\DependencyInjection\Compiler;
+
+use Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler\ControllerArgumentValueResolverPass;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\HttpKernel\Controller\ArgumentResolver;
+
+class ControllerArgumentValueResolverPassTest extends \PHPUnit_Framework_TestCase
+{
+    public function testServicesAreOrderedAccordingToPriority()
+    {
+        $services = array(
+            'n3' => array('tag' => array()),
+            'n1' => array('tag' => array('priority' => 200)),
+            'n2' => array('tag' => array('priority' => 100)),
+        );
+
+        $expected = array(
+            new Reference('n1'),
+            new Reference('n2'),
+            new Reference('n3'),
+        );
+
+        $definition = new Definition(ArgumentResolver::class, array(null, array()));
+        $container = $this->getMock(ContainerBuilder::class, array('findTaggedServiceIds'));
```

Comment:
You can just create a `ContainerBuilder` instance here (like, for example, done in the `FormPassTest` and `CachePoolPassTest`).